src/ml-service/packages/sensor_location/locate/modules/locate_model.py

```diff
@@ -0,0 +1,116 @@
+import pandas as pd
+from pymongo import MongoClient
+import pymongo
```

Comment:
Could be great to just have a models directory which defines the schemas (represented as separate files) of our collections as seen in ATLAS:
1. locate
2. geo_census
3. geometry_polygon
4. locate_map
5. locatemap

Taking an example of `locatemap` below:
```
from .db import db

user_id: Object
space_name: String
plan: Object

class locatemap(db.Document):
    user_id = db.StringField(required=True, unique=True)
    space_name= db.ListField(db.StringField(), required=True)
    plan = db.ObjectField(db.StringField(), required=True)
```